inline-r/src/Language/R/QQ.hs

```diff
@@ -54,6 +54,7 @@ import qualified System.IO.Temp as Temp
 import System.Process
 import System.IO
 import System.Exit
+import Debug.Trace
```

Comment:
Is this necessary, or a remnant from debugging?

Reply:
Remnant from debugging